docs/fsharp/language-reference/members/constructors.md

```diff
@@ -82,9 +82,11 @@ The following version of the previous code illustrates the combination of ordina
 
 [!code-fsharp[Main](../../../../samples/snippets/fsharp/lang-ref-2/snippet3507.fs)]
     
+## Constructors in inherited class
+You can definition constructors combined inherited class. For more information, see [Constructors and inheritance](../inheritance.md#constructors-and-inheritance).
```

Comment:
> You can definition constructors combined inherited class.

I don't understand what is this trying to say. I'm assuming "definition" should have been "define". But I don't get what does "combined" mean here.